cg/test/blockchain-setup.js

```diff
@@ -5,11 +5,10 @@ const ContractService = require('../src/domains/management/contract/contract.ser
506001908051906020019061112a9291906115f7565b506000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff166001600081548110151561115d57fe5b9060005260206000200160006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff160217905550600091505b835182101561132a576000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff1684838151811015156111f957fe5b9060200190602002015173ffffffffffffffffffffffffffffffffffffffff161415151561122657600080fd5b6001820190505b83518110156112ac57838181518110151561124457fe5b9060200190602002015173ffffffffffffffffffffffffffffffffffffffff16848381518110151561127257fe5b9060200190602002015173ffffffffffffffffffffffffffffffffffffffff161415151561129f57600080fd5b808060010191505061122d565b83828151811015156112ba57fe5b9060200190602002015160018084018154811015156112d557fe5b9060005260206000200160006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555081806001019250506111aa565b600192505050919050565b600080600090505b825181101561137e57611366838281518110151561135757fe5b90602001906020020151610702565b151561137157600080fd5b808060010191505061133d565b6001915050919050565b600060026000836000191660001916815260200190815260200160002060000160009054906101000a900460ff169050919050565b60608060008060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561141d57600080fd5b836000151560026000836000191660001916815260200190815260200160002060000160009054906101000a900460ff16151514151561145c57600080fd5b6001805490506040519080825280602002602001820160405280156114905781602001602082028038833980820191505090505b509250600091505b6001805490508210156115885760026000866000191660001916815260200190815260200160002060020160006001848154811015156114d457fe5b9060005260206000200160009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060009054906101000a900460ff16838381518110151561155557fe5b90602001906020020190600281111561156a57fe5b9081600281111561157757fe5b815250508180600101925050611498565b7f5cee2268933cd873c783419e5be7668e121517767fe669a5a703b09426b138158560405180826000191660001916815260200191505060405180910390a1829350505050919050565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b828054828255906000526020600020908101928215611670579160200282015b8281111561166f5782518260006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555091602001919060010190611617565b5b50905061167d9190611681565b5090565b6116c191905b808211156116bd57600081816101000a81549073ffffffffffffffffffffffffffffffffffffffff021916905550600101611687565b5090565b905600a165627a7a723058200ad03be1f5d4bf702118989ff2ff8bc1cbb12b05d9e49a089fcbd6869a300a950029';
```

Comment:
Maybe add an issue to have this only in one place – very error prone the way it is now..

Reply:
Agree